### ISSUE TITLE
running IJulia with a local project environment

### ISSUE THREAD

**Issue by joachimneu** [JuliaLang/IJulia.jl#750]
I use IJulia with Julia v1.0.0 on Linux. I have IJulia installed only in the project-local environment, rather than the v1.0 "global" environment (see https://docs.julialang.org/en/v1/stdlib/Pkg/index.html#Glossary-1). In that case, the Jupyter kernel does not start properly (observed behavior very similar to #693 but turned out to be unrelated). After turning on Jupyter debug output, I see:

```
[I 01:55:46.000 NotebookApp] KernelRestarter: restarting kernel (1/5), new random ports
[D 01:55:46.002 NotebookApp] Starting kernel: ['/home/.../bin/julia-1.0.0/bin/julia', '-i', '--startup-file=yes', '--color=yes', '/home/.../.julia/packages/IJulia/4VL8h/src/krnel.jl', '/run/user/.../jupyter/kernel-....json']
[D 01:55:46.010 NotebookApp] Connecting to: tcp://127.0.0.1:55337
ERROR: LoadError: ArgumentError: Package IJulia not found in current path:
- Run `Pkg.add("IJulia")` to install the IJulia package.

Stacktrace:
 [1] require(::Module, ::Symbol) at ./loading.jl:817
 [2] include at ./boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] exec_options(::Base.JLOptions) at ./client.jl:229
 [6] _start() at ./client.jl:421
in expression starting at /home/.../.julia/packages/IJulia/4VL8h/src/kernel.jl:1
[I 01:55:49.013 NotebookApp] KernelRestarter: restarting kernel (2/5), new random ports
[D 01:55:49.016 NotebookApp] Starting kernel: ['/home/.../bin/julia-1.0.0/bin/julia', '-i', '--startup-file=yes', '--color=yes', '/home/.../.julia/packages/IJulia/4VL8h/src/krnel.jl', '/run/user/.../jupyter/kernel-....json']
```

Turns out if I install IJulia also in the v1.0 "global" environment, it works.

**Comment by stevengj** [comment]
Jupyter launches IJulia with basically `julia kernel.jl` … your local project is not activated there, I think?

**Comment by joachimneu** [comment]
Ok, now I understand why this is happening and was surprising to me. Before Julia shipped its own "built-in" environments, I used virtualenv + Playground.jl. Both of which remained active for newly spawned Jupyter/Julia processes, and hence explicit activation of the environment inside IJulia was not necessary.

**Comment by joachimneu** [comment]
How about adding a remark like this to the readme?

**Comment by stevengj** [comment]
Since creating local projects is a somewhat specialized thing in Julia (most users don't do it right now), I would put it later in the README.   It would also be nice to note that you can install a kernel that launches into a particular project by passing `-e 'import Pkg; Pkg.activate("myproject")' -L /path/to/kernel.jl`, maybe with a patch to make this easier.

**Comment by stevengj** [comment]
Fixed by #820.
